app/services/pipeline/storage/rubric_service.rb

```diff
@@ -7,18 +7,68 @@ class RubricService
       def self.call(context:)
         Rails.logger.info("Storing rubric for assignment: #{context.assignment.id}")
 
-        # This would contain actual persistence logic in a real implementation
-        # Following PORO principles - keep it simple and focused
-        context.rubric = create_rubric(context)
+        context.rubric = create_rubric_with_criteria_and_levels(context)
         context
       end
 
       private
 
+      def self.create_rubric_with_criteria_and_levels(context)
```

Comment:
This method should be renamed since it is not actually creating a rubric. It is attaching new criteria and levels to an existing rubric.